### PR TITLE
[table] upgrade the handle to 32 bytes

### PR DESCRIPTION
**Pull request by zekun000**
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3212)

### CODE REVIEW

File: aptos-move/aptos-aggregator/src/aggregator_extension.rs [hunk 1]
```diff
@@ -413,12 +415,12 @@ mod test {
...
 
     fn test_id(key: u128) -> AggregatorID {
-        AggregatorID::new(0, key)
+        AggregatorID::new(TableHandle(AccountAddress::ZERO), key)
     }
 
     fn id_to_state_key(id: AggregatorID) -> StateKey {
         let key_bytes = serialize(&id.key);
-        StateKey::table_item(AptosTableHandle(id.handle), key_bytes)
+        StateKey::table_item(AptosTableHandle::from(id.handle), key_bytes)
```
Comment (msmouse) [inline aptos-move/aptos-aggregator/src/aggregator_extension.rs:423]: why not do `.into()`?

**Review by msmouse — APPROVED** [review]

File: aptos-move/aptos-aggregator/src/aggregator_extension.rs (hunk 1 above)
Comment (zekun000) [reply aptos-move/aptos-aggregator/src/aggregator_extension.rs:423]: this seems more explicit

Comment (msmouse) [reply aptos-move/aptos-aggregator/src/aggregator_extension.rs:423]: 🤷‍♂️

**Comment by vgao1996** [comment]
@zekun000 it looks like you'll need to run these commands to update some api spec?
```
cargo run -p aptos-openapi-spec-generator -- -f yaml -o api/doc/spec.yaml
cargo run -p aptos-openapi-spec-generator -- -f json -o api/doc/spec.json
```

**Review by wrwg — APPROVED** [review]

**Pull request merged**